Update Google Cast docs (#17549)
home-assistant/home-assistant.github.io: source/_integrations/cast.markdown

        - url: "https://tilos.hu/images/kockalogo.png"
```

> ## Trouble shooting automatic discovery
> mDNS relies on UDP multicast, which may fail for various reasons. If none of the tips in this section helps, the recommended solution is to ensure all cast devices have static IPs assigned to them and configure a list of known hosts.

Reviewer: ```suggestion
## Troubleshooting automatic discovery

mDNS relies on UDP multicast, which may fail for various reasons. If none of the tips in this section helps, the recommended solution is to ensure all cast devices have static IPs assigned to them and configure a list of known hosts.
```

> ### Zeroconf configuration
> The Google Cast integration relies on the [Zeroconf integration](/integrations/zeroconf) for mDNS discovery. The Zeroconf integration has some configuration options which impact mDNS routing.

Reviewer: ```suggestion
### Zeroconf configuration

The Google Cast integration relies on the [Zeroconf integration](/integrations/zeroconf) for mDNS discovery. The Zeroconf integration has some configuration options which impact mDNS routing.
```

### Cast devices and Home Assistant on different subnets

Cast devices can only be automatically discovered if they are on the same subnet as Home Assistant because mDNS packets are not routed across subnets.

> Automatic discovery in setups with cast devices on a different subnet than Home Assistant are not recommended and not supported.

Reviewer: ```suggestion
Automatic discovery in setups with cast devices on a different subnet than Home Assistant is not recommended and not supported.
```
